### ISSUE TITLE
XCode error when archiving

### ISSUE THREAD

**Issue by LeonardoDemartino** [Purchasely/Unity-Package#24]
This error seems similar to #12:

```
error: Multiple commands produce 'Xcode/DerivedData/Unity-iPhone-cyldpxtghqwzpdccszapfqhzkfvr/ArchiveIntermediates/Unity-iPhone/InstallationBuildProductsLocation/Applications/[APP].app/Frameworks/Purchasely.framework'
    note: Target 'Unity-iPhone' (project 'Unity-iPhone') has copy command from '/Pods/Purchasely/Purchasely/Frameworks/Purchasely.xcframework/ios-arm64/Purchasely.framework' to '/Developer/Xcode/DerivedData/Unity-iPhone-cyldpxtghqwzpdccszapfqhzkfvr/ArchiveIntermediates/Unity-iPhone/InstallationBuildProductsLocation/Applications/BoopKids.app/Frameworks/Purchasely.framework'
    note: That command depends on command in Target 'Unity-iPhone' (project 'Unity-iPhone'): script phase “[CP] Embed Pods Frameworks”
```

It seems that removing Purchasely.xcframework from "Embed Framework" phase fixed the issue.

**Comment by kherembourg** [comment]
Hello @LeonardoDemartino @dominik-figlak-cpg 

Does it work for you if you comment this line?
https://github.com/Purchasely/Unity-Package/blob/3847d1caf465678af8747bd13b052599d313d75c/Editor/PurchaselyPostProcessor.cs#L27

Can I ask how you add this SDK to your project? Do you use the package manager and add the dependency from github?

Thanks

**Comment by dominik-figlak-cpg** [comment]
Hello @kherembourg

For me comment this line works and looks like build works properly with Purchasely dependency. We added SDK by downloaded code from github because we already needed to disable the Android part in the Purchasely post processor script due to the fact that our build.gradle is large and has more than one dependencies section, which is why your dependencies were added in the wrong place and the build for Android was not built earlier. We add these dependencies to gradle manually

**Comment by LeonardoDemartino** [comment]
@kherembourg Yes, I use the package manager with the dependency from the github url.

**Comment by EPIKorial** [comment]
We have pinpointed the issue's origin, addressed it, and the solution will be available in our upcoming release. Thank you for bringing this matter to our attention, and we appreciate your collaboration.
